notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1276_star_selector_pointing_tests.ipynb

```diff
@@ -0,0 +1,231 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d5066b70-de7b-4618-b0f4-746e029f8ebc",
+   "metadata": {},
+   "source": [
+    "## SITCOM-1276: Notebook to find pairs of stars for star-tracker tests with slew and settle¶\n",
+    "\n",
+    "This notebook will produce a list of pairs of stars brighter than Vmag_max, at a given UTC t0, separated by an angular range between sep_lo and sep_hi, and higher than elevation elevation_min.\n",
+    "\n",
+    "This is to produce these positions manually. This notebook will be followed up by script that produces a file with the positions and write a json BLOCK for the corresponding observations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3098bdeb-30c9-4818-86ee-11ea1ca826f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import astropy.units as u\n",
+    "from astropy.coordinates import AltAz, EarthLocation, SkyCoord\n",
+    "from astropy.time import Time\n",
+    "from astroquery.vizier import Vizier"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6f6a9f9c-50b2-4019-a3cb-813b666763f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define some configuration variables\n",
+    "Vmag_max = 5 # maximum magnitude of the stars in the list\n",
+    "t0 = '2024-3-8 01:00:00' #in UTC, ideally the central time at which observations will take place\n",
+    "sep_lo = 3.3 #in degrees, minimum separation between stars\n",
+    "sep_hi = 3.7 #in degrees, maximum separation between stars\n",
+    "elevation_min = 45. #  minimum elevation for stars in the pair, NOTE that some might go below\n",
+    "                    # this elevation threshold as the sky rotates during the BLOCK"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "002fa1f5-4ecf-4dd7-86c5-054496185fe8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# select the site\n",
+    "pachon_site = EarthLocation(lat=-30.2407216*u.deg,lon=-70.736583*u.deg,height=2700*u.m)"
```

Comment:
I think astropy and astroquery (I know at astroplan does) would allow you to select Rubin by name.